libraries/AP_Camera/AP_Camera_Params.cpp

```diff
@@ -81,6 +81,12 @@ const AP_Param::GroupInfo AP_Camera_Params::var_info[] = {
     // @User: Standard
     AP_GROUPINFO("_OPTIONS", 10, AP_Camera_Params, options, 0),
 
+    // @Param: _MNT_INST
+    // @DisplayName: Mount instance
+    // @Description: Mount instace to which camera is associated with
```

Comment:
Let's add something like this to the end, "If zero the camera instance number is used.  E.g. camera1 is assumed to be in mount1, etc."